### ISSUE TITLE
Visual Accessibility option for Svelte API docs

### ISSUE THREAD

**Issue by subz390** [sveltejs/sites#105]
**Is your feature request related to a problem? Please describe.**
Yes, it's related to Web Content Accessibility, the visual theme of your [website documentation](https://svelte.dev/docs).

**Describe the solution you'd like**
Do you plan or would consider including Visual Accessibility options in the future?
I have ideas for an Accessibility component for Svelte in addition to general discussions about themes sveltejs/svelte#1550

**Describe alternatives you've considered**
I've started work on a **dark mode** theme that I am happy to share with you.
![190912-0732 accessibility](https://user-images.githubusercontent.com/11546815/64760461-a29fb480-d531-11e9-83d9-db464eb7fd72.png)

**How important is this feature to you?**
I cannot reasonably accesses your website for a prolonged period of time without visual adaptation, this means I have limited access to the resources contained therein, an adjustment to the theme restores access.

**Additional context**
It's recognised in today's day and age that content and service providers take reasonable steps to provide equal access.  But it's more than that, the Svelte website is _the_ Ambassador of Svelte and a well thought out visual accessibility solution is just another super way to say good things about the ecosystem.  Here is the [Web Content Accessibility Guidelines (WCAG) 2.1](https://www.w3.org/TR/WCAG21/).

**Comment by NewUserHa** [comment]
the color(the background, font color) is too green in your demo, which make ppl very uncomfortable.
we should learn vscode dark or youtube dark, or windows dark, all of them has good color on all different monitors.

**Comment by subz390** [comment]
> the color(the background, font color) is too green in your demo, which make ppl very uncomfortable. we should learn vscode dark or youtube dark, or windows dark, all of them has good color on all different monitors.

Great suggestion.  You may freely fork my repo, edit the colors that make you feel uncomfortable, then use the CSS in Stylish or as a userscript, so many choices, and don't forget to link your project back here so other people can benefit from your input.

**Comment by benmccann** [comment]
we have dark mode on the site now. please file at https://github.com/sveltejs/svelte.dev/issues/new if there is more to be done
